modifiers/heading_id.js

```diff
@@ -110,6 +111,20 @@ module.exports = (md, options) => {
             token.attrs.push(idAttr);
         }
 
+        // Generate hash link if option is set
+        if (optsObj.hashLink && level <= 3) {
```

Comment:
Could we add a `hashLinkLevel` setting that defaults to `3`, for if a user wanted to show hash links on all headings?